src/data/mock/data/reisetilskudd.ts

```diff
@@ -0,0 +1,219 @@
+import { RSSoknad } from '../../../types/rs-types/rs-soknad'
+import { arbeidstakerReisetilskudd } from './sykmeldinger'
+
+export const nyttReisetilskudd: RSSoknad = {
+    'arbeidssituasjon': 'ARBEIDSTAKER',
+    'egenmeldtSykmelding': false,
+    'korrigerer': null,
+    'korrigertAv': null,
+    'sendtTilArbeidsgiverDato': null,
+    'soknadPerioder': [ {
+        'fom': '2021-02-01',
+        'tom': '2021-02-18',
+        'grad': 0,
+        'sykmeldingstype': 'REISETILSKUDD'
+    } ],
+    'soknadstype': 'REISETILSKUDD',
+    'startSykeforlop': '2021-02-01',
+    'sykmeldingUtskrevet': '2021-02-01',
+    'id': '3b6d3764-bc4d-4fe2-902d-5097b9e0ce93',
+    'status': 'NY',
+    'sykmeldingId': arbeidstakerReisetilskudd.id,
+    // 'fnr': '01010112345',    TODO: Trenger vi dette?
+    'fom': '2021-02-01',
+    'tom': '2021-02-18',
+    'opprettetDato': '2021-02-19',
+    // 'endret': '2021-02-19T10:25:59.232720Z', TODO: Trenger vi dette?
+    'sendtTilNAVDato': null,
+    'avbruttDato': null,
+    'arbeidsgiver': { 'navn': 'BYGDA SFO', 'orgnummer': '995816598' },  // TODO: Slått sammen til ett arbeidsgiverobjekt
+    'sporsmal': [
+        {
+            'id': '7d293cf9-6274-4d05-80f3-f7a6be337506',
+            'tag': 'ANSVARSERKLARING',              // TODO: Skal denne være lik som sykepengesoknad sin førsteside?
```

Comment:
Stemmer for lik som de andre inntill videre

Reply:
Ja, skal jeg også legge til spørsmålene VAER_KLAR_OVER_AT og BEKREFT_OPPLYSNINGER, sånn at det blir likt på slutten av søknaden

Reply:
Ja 👍